solutions/landing-zone/README.md

```diff
@@ -102,21 +102,11 @@ To deploy this Landing Zone you will first need to create a Bootstrap project wi
 
 1. Deploy Bootstrap
 
-    ```
-    arete create landing-zone-controller --region=northamerica-northeast1
-    ```
+    This bootstrap assume you have a [Config Controller](https://cloud.google.com/anthos-config-management/docs/concepts/config-controller-overview) instance provisioned already. If you do not you can follow either the [quickstart]https://cloud.google.com/anthos-config-management/docs/concepts/config-controller-overview) guide or the [detailed install guide](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/docs/advanced-install.md) for advanced users.
```

Comment:
better, for a new user they would have stood up a kcc cluster and then had the instructions to setup another one in the solution.

I recommend for advanced users to directly to the advanced install and they will know to skip the arete step for cc cluster creation in the solution

Reply:
Updated Wording